geopandas_view/view.py

```diff
@@ -488,14 +490,35 @@ def _choropleth(
         if classification_kwds is None:
             classification_kwds = {}
         if "k" not in classification_kwds:
-            classification_kwds["k"] = bins
+            classification_kwds["k"] = k
 
         binning = mapclassify.classify(
             np.asarray(gdf[column]), scheme, **classification_kwds
         )
         bins = binning.bins.tolist()
         bins.insert(0, gdf[column].min())
 
+    else:
+
+        vmin = gdf[column].min() if not vmin else vmin
+        vmax = gdf[column].max() if not vmax else vmax
+        if vmin > gdf[column].min():
+            warn(
```

Comment:
Shall we warn or raise here? warn is more permissive but we may want to be strict. Don't know her.e